controllers/auth_policy_status_updater.go

```diff
@@ -143,7 +143,10 @@ func (r *AuthPolicyStatusUpdater) enforcedCondition(policy *kuadrantv1.AuthPolic
 		if len(kuadrantv1.PoliciesInPath(effectivePolicy.Path, func(p machinery.Policy) bool { return p.GetLocator() == policy.GetLocator() })) == 0 {
 			continue
 		}
-		gatewayClass, gateway, listener, httpRoute, httpRouteRule, _ := kuadrantpolicymachinery.ObjectsInRequestPath(effectivePolicy.Path)
+		gatewayClass, gateway, listener, httpRoute, httpRouteRule, err := kuadrantpolicymachinery.ObjectsInRequestPath(effectivePolicy.Path)
+		if err != nil {
+			continue
```

Comment:
I worry a little when I see this. Why do we want to swallow the error here?

Reply:
as this only currently ever returns an invalid path error I think we should log them at a debug level to help understand what might be happening in logs same for below

Reply:
Updated with debug info log